lib/house.rb

```diff
@@ -0,0 +1,58 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms,
+              :house_details
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @house_details = Hash.new
+  end
+
+  def price
+    @price.delete("$").to_i
```

Comment:
I like that you are formatting the price correctly to be an integer. One thing that is slightly confusing is you both have the method here `#price` and the `attr_reader :price`. This seems to be a bit redundant as under the hood when we use attr_reader ruby created a method `price` for us automatically.

If you want to format the price as it comes to intialize method I would do it there